test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1701,6 +1701,54 @@ def check_number_of_cooling_towers(prototypes_base)
     end
   end
 
+
+  
+ # Check if coefficients of part-load power curve is correct per G3.1.3.15
+  def check_variable_speed_fan_power(prototypes_base)
+    prototypes_base.each do |prototype, model|
+      model.getFanVariableVolumes.each do |supply_fan|
+        supply_fan_name = supply_fan.name.get.to_s
+
+      
+        # check fan curves
+        # Skip single-zone VAV fans
+        next if supply_fan.airLoopHVAC.get.thermalZones.size == 1
+        # coefficient 1
+        if supply_fan.fanPowerCoefficient1.is_initialized
+          expected_coefficient = 0.0013
+          coefficient = supply_fan.fanPowerCoefficient1.get
+          assert((coefficient - expected_coefficient).abs < 0.01, "Expected Coefficient 1 for #{supply_fan_name} to be equal to #{expected_coefficient}; found #{coefficient} instead")
```

Comment:
The test tolerance of 0.01 is too large. You could fix it by doing a percentage calculation:
((coefficient - expected_coefficient)/expected_coefficient).abs < 0.01